### PR TITLE
display real contacts on profile; minor refactor on contacts

### PR DESCRIPTION
**Pull request by iurimatias**

### CODE REVIEW

File: src/signals/messages.nim [hunk 1]
```diff
@@ -83,6 +83,7 @@ proc toChat*(jsonChat: JsonNode): Chat =
       result.members.add(jsonMember.toChatMember)
 
 proc toMessage*(jsonMsg: JsonNode): Message =
+  echo $jsonMsg
```
Comment (PascalPrecht) [inline src/signals/messages.nim]: left over?

File: src/status/contacts.nim [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+type
+  Contact* = ref object
+    name*, address*, identicon*: string
+
+proc toContact*(self: JsonNode): Contact =
+  result = Contact(
+    name: self["alias"].getStr,
+    address: self["id"].getStr,
+    identicon: self["identicon"].getStr
+  )
```
Comment (PascalPrecht) [inline src/status/contacts.nim:15]: We're having `Profile` and `Contact` types. Might make sense to unify those?

Comment (iurimatias) [reply src/status/contacts.nim:15]: good point, will do this

File: ui/app/AppLayouts/Profile/Sections/Contacts/Contact.qml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }
+    Text {
+        id: addressText
+        width: 108
+        text: address
+        elide: Text.ElideMiddle
+        anchors.bottom: accountImage.bottom
+        anchors.bottomMargin: 0
+        anchors.left: usernameText.left
+        anchors.leftMargin: 0
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Contacts/Contact.qml:45]: Just an FYI, you can remove the margins that are `0`, since it's the default

**Pull request merged**